make/conf/jib-profiles.js

```diff
@@ -1275,6 +1275,15 @@ var getJibProfilesDependencies = function (input, common) {
             ext: "tar.gz",
             revision: "3.4.2+1.0"
         },
+        tidy: {
+            organization: common.organization,
+            environment_name: "TIDY",
+            ext: "tar.gz",
+            revision: "5.9.20+1",
+            environment_path: input.get("tidy", "home_path") + "/tidy/bin/tidy",
+            configure_args: "TIDY=" + input.get("tidy", "home_path") +"/bin/tidy",
```

Comment:
These paths are different. I'm guessing the latter one is corret?

Reply:
Yes, the second one is correct. Will fix it.